determined-ai/determined
# chore: replace `enum` with `object as const`

## Description
[WEB-534]



## Test Plan
- Check if all are replaced with `as const`
- Check of everything works as expected



## Commentary (optional)
- Type is more strict now which is amazing!
- I left [these enums](https://github.com/determined-ai/determined/blob/3f7196841e1183e19a9f671bca44126aca83c572/webui/react/src/shared/utils/service.test.ts#L2) as it is on purpose cuz these mock `api.ts`



## Checklist
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.





[WEB-534]: https://determinedai.atlassian.net/browse/WEB-534?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
